### DISCUSSION TITLE
Why does the volume.tier.move operation not apply the toReplication parameter?

### DISCUSSION THREAD

**Discussion by photosssa** [seaweedfs/seaweedfs#6313] [General]
I have a configuration of 2 SSDs and 2 HDDs, and I use fs.configure to direct all writes to the SSDs. I will move the newly written volumes to the HDDs at the appropriate time. Although I specified the toReplication=001 parameter, I find that when the operation is completed, the volume on the SSD has been deleted, and there is only a single copy of that volume on the HDD. I need an additional volume.fix.replication or directly run ec.encode to restore its redundancy. During the time when there is only a single copy, does this not increase the risk of data loss? Why doesn't the volume.tier.move operation directly apply the toReplication parameter to reduce this risk?
```
volume.tier.move -fromDiskType=ssd -toDiskType=hdd -fullPercent=2 -quietFor=0s -toReplication=001 -force
```

**Comment by zagrider** [comment]
I'm encountering the same issue, when you trigger volume.tier.move command, there's no copy of the volume moved instead of two if you selected 001 or set the replication to 010 with the volume.tier.move command. I need to do a volume.fix.replication. it'll would be great to keep the redundancy in the process, in case of HDD failure in the process. Especially for those using several HDDs in RAID0 mode.
